### PR TITLE
Initial project setup

### PR DESCRIPTION
**Pull request by AndrewSakovich**

### CODE REVIEW

File: src/navigators/nameScreen.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export enum nameScreen {
```
Comment (evgenyshenets91) [inline src/navigators/nameScreen.ts]: enum should starts from Upper case letter

File: src/redux/reducers/TodoReducer.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+const initialState: TodoReducerState = {
+  todoItems: [],
+  userToken: null,
+  user: null,
+};
+
+export const todoReducer = (
+  state = initialState,
+  action: any,
```
Comment (evgenyshenets91) [inline src/redux/reducers/TodoReducer.ts:22]: any not necessary

File: src/redux/reducers/TodoReducer.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      return {
+        ...state,
+        todoItems: [...state.todoItems, newItem],
+      };
+    }
+    case TodoActionTypes.DELETE_ITEM: {
+      const { id }: ItemDeletePayload = action.payload;
+
+      const newTodoItems = [
+        ...state.todoItems.filter(item => {
```
Comment (evgenyshenets91) [inline src/redux/reducers/TodoReducer.ts]: we already discussed it, why did you skip it?

File: src/redux/store.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { createStore } from 'redux';
+import { todoReducer, TodoReducerState } from './reducers/TodoReducer';
+import { persistStore, persistReducer } from 'redux-persist';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+// defaults to localStorage for web
```
Comment (evgenyshenets91) [inline src/redux/store.ts]: ?

File: src/screens/ItemListScreen/index.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  const data = useSelector<ReduxStoreType, TodoItemType[]>(
+    doneItemsSelectors(flagDone),
+  );
+
+  const renderItem: ListRenderItem<TodoItemType> = ({ item }) => {
+    return <TodoItem todoItem={item} />;
+  };
+
+  return (
+    <View style={style.container}>
```
Comment (evgenyshenets91) [inline src/screens/ItemListScreen/index.tsx]: I am not sure that we have to add here view, we can make necessary styles by using Flatlist

File: src/screens/LoginScreen/index.tsx [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { View } from 'react-native';
+import { style } from './style';
+import { SignInButton } from '../../components/SignInBotton';
+
+export const LoginScreen: FC = () => {
```
Comment (evgenyshenets91) [inline src/screens/LoginScreen/index.tsx:10]: all logic, related signIn and dispatch actions we have to pass inside LoginScreen,button will get props.

File: src/navigators/nameScreen.ts (hunk 1 above)
Comment (AndrewSakovich) [reply src/navigators/nameScreen.ts]: got it, i will fixed

File: src/redux/reducers/TodoReducer.ts (hunk 2 above)
Comment (AndrewSakovich) [reply src/redux/reducers/TodoReducer.ts:22]: if I remove this type I get an error:  Parameter 'action' implicitly has an 'any' type.

File: src/redux/reducers/TodoReducer.ts (hunk 3 above)
Comment (AndrewSakovich) [reply src/redux/reducers/TodoReducer.ts]: sory, I will fix it

File: src/redux/store.ts (hunk 4 above)
Comment (AndrewSakovich) [reply src/redux/store.ts]: I will deleted this coment

File: src/screens/ItemListScreen/index.tsx (hunk 5 above)
Comment (AndrewSakovich) [reply src/screens/ItemListScreen/index.tsx]: ok, i agree, i will fix it

File: src/screens/LoginScreen/index.tsx (hunk 6 above)
Comment (AndrewSakovich) [reply src/screens/LoginScreen/index.tsx:10]: yes it's my mistake

**Pull request merged**